course3/UniquePtr/Makefile

```diff
@@ -0,0 +1,17 @@
+# Makefile for MyUniquePtr and test
```

Comment:
It's good that the noble art of crafting Makefiles is not lost, but I would invest some energy in mastering CMake